docs/content/quickstart/desired-state.md

```diff
@@ -255,12 +256,34 @@ The installation is out-of-sync, running the upgrade action...
 Skipping bundle execution because --dry-run was specified
 ```
 
-## Cleanup
+## Uninstall
 
-To clean up the resources installed from this QuickStart, use the `porter uninstall` command.
+Installations have a field named **active** that control if the installation should be installed and managed by Porter.
+After a bundle has been installed, set active to false on the installation to uninstall it.
 
+Edit installation.yaml, set active to false, and then save the file.
+
+```yaml
+active: false
```

Comment:
I wonder if it would be helpful to have separate config values for controlling `Uninstall` and `Managed Install`. The default behavior would be auto-installation and not uninstall. I feel having them being separate helps me understand what each of the config values mean
```yaml
autoInstall: false
uninstall: false
```

Reply:
What I am having trouble with when it comes to naming the field is that the installation document is intended to represent the desired state of the installation, and shouldn't contain fields that represent a specific action. I originally thought to go with something like `uninstalled` but felt it didn't fit well with that approach.

I'm not sure what `dry-run` is supposed to indicate? Is that for when you create an installation that has active=false, which causes porter to just ignore the installation entirely?

Reply:
I could be persuaded to use uninstalled though. I think we could make that work and remove the oddball case of "making a new installation with active=false should do nothing" which is a scenario I'm not sure we care to actually support. It would be pretty weird for someone to make a new installation and set "uninstalled" to true, since something can't be uninstalled until it's installed first.

I'm confusing myself! 😝

Reply:
> I'm not sure what dry-run is supposed to indicate? Is that for when you create an installation that has active=false, which causes porter to just ignore the installation entirely?

Yes, that's exactly what I meant by `dry-run`. My thoughts on the use case for this is maybe someone want to use the installation document to manage the state of an install action, instead of the `--dry-run` flag. But I agree that this is a somewhat odd case. 


In terms of the `uninstalled` state, my thinking is that if there's nothing installed then that means it's already the desired state since that's the result of `uninstalled: true`

Reply:
So for uninstalled, it would default to false correct? If I apply an installation that doesn't exist yet with uninstalled=true, would porter do nothing? Store the record? That is how it works in this PR currently, and I'm trying to understand if we just want to rename active to uninstalled or if the behavior needs to change too.

Reply:
To be clear I really don't like that right now active defaults to false, it kind of goes against how Porter does things, so this is probably the way we want to go. I'm just making sure I understand your feedback. 😀

Reply:
I'm going to tweak this PR to use "uninstalled" instead of "active". If someone makes an installation with uninstalled = true, then we will not install the bundle.